orte/mca/ess/base/ess_base_std_orted.c

```diff
@@ -499,26 +526,27 @@ int orte_ess_base_orted_setup(char **hosts)
 
     /* setup the PMIx framework - ensure it skips all non-PMIx components,
      * but do not override anything we were given */
-    opal_setenv("OMPI_MCA_pmix", "^s1,s2,cray,isolated", false, &environ);
-    if (OPAL_SUCCESS != (ret = mca_base_framework_open(&opal_pmix_base_framework, 0))) {
-        ORTE_ERROR_LOG(ret);
-        error = "orte_pmix_base_open";
-        goto error;
-    }
-    if (ORTE_SUCCESS != (ret = opal_pmix_base_select())) {
-        ORTE_ERROR_LOG(ret);
-        error = "opal_pmix_base_select";
-        goto error;
-    }
+//[A]
```

Comment:
Please remove old code rather than commenting out large blocks.  If the code is needed again, it can be pulled out of history.